tests/phpunit/api/v3/PaymentProcessorTest.php

```diff
@@ -134,6 +138,7 @@ public function testPaymentProcessorDelete() {
 
   /**
    * Check with valid params array.
+   * @dataProvider versionThreeAndFour
```

Comment:
For this test file - note that it's not sufficient to add the `@dataProvider` annotation, you must also add the `$version` param to the function and then the first line add `$this->_apiversion = $version;`

Reply:
Fixed